tests/test_A0_initialization.py

```diff
@@ -8,79 +8,92 @@
-            )
-
-    def test_user_input_path_input_file_not_existing(self):
-        with pytest.raises(FileNotFoundError):
-            initializing.get_user_input(
-                path_input_file=os.path.join(self.input_path, "not_existing.json"),
-                path_output_folder=self.output_path,
-                overwrite=True,
-            )
+    def test_input_folder_is_copied_in_output_within_folder_named_input(self):
+        pass
+        # initializing.get_user_input(path_output_folder=self.output_path)
+        # assert os.path.exists(self.output_path)
+
+    def test_input_folder_not_existing_raise_filenotfound_error(self):
+        pass
+        # with pytest.raises(NotADirectoryError):
+        #     initializing.get_user_input(
+        #         path_input_file=self.fake_input_path,
+        #         path_output_folder=self.output_path,
+        #     )
+
+    def test_if_json_opt_and_no_json_file_in_input_folder_raise_filenotfound_error(
+        self,
+    ):
+        pass
+        # with pytest.raises(FileNotFoundError):
+        #     initializing.get_user_input(
+        #         path_input_file=os.path.join(self.input_path, "not_existing.json"),
+        #         path_output_folder=self.output_path,
+        #         overwrite=True,
+        #     )
+
+    def test_if_json_opt_and_more_than_one_json_file_in_input_folder_raise_fileexists_error(
+        self,
+    ):
+        pass
+        # with pytest.raises(FileNotFoundError):
+        #     initializing.get_user_input(
+        #         path_input_file=os.path.join(self.input_path, "not_existing.json"),
+        #         path_output_folder=self.output_path,
+        #         overwrite=True,
+        #     )
+
+    def test_if_json_opt_path_input_file_set_to_path_input_folder(self):
+        """Check that the path_input_file is <path_input_folder>
+
+            in module A1_csv_to_json this will be transformed into a .json file
+        """
+        pass
+
+    def test_if_csv_opt_and_csv_elements_folder_not_in_input_folder_raise_filenotfound_error(
+        self,
+    ):
+        pass
+        # with pytest.raises(FileNotFoundError):
+        #     initializing.get_user_input(
+        #         path_input_file=os.path.join(self.input_path, "not_existing.json"),
+        #         path_output_folder=self.output_path,
+        #         overwrite=True,
+        #     )
+
+    def test_if_csv_opt_path_input_file_set_to_path_input_folder_mvs_csv_config_dot_json(
+        self,
+    ):
+        """Check that the path_input_file is <path_input_folder>/mvs_csv_config.json """
+        pass
+
+    def test_if_log_opt_display_output_is_set_with_correct_value(self):
+        pass
+
+    def test_if_path_output_folder_exists_raise_fileexists_error(self):
+        """The error message should advice the user to use -f option to force overwrite"""
+        pass
+
+    def test_if_f_opt_preexisting_path_output_folder_should_be_replaced(self):
```

Comment:
"should be replaced" as in test if user dialog appear or as in if folder is replaced or not?

Reply:
as in "if folder is replaced or not"

I would suggest to not have user dialogs: 
-if the option -f is not used we warn that the file exists and the user can either force the overwrite with -f or rename the output folder with a different name
- if the option -f is used, the user is aware the folder will be overwritten

I find it less cumbersome to code and test

Reply:
I agree, people using -f should know what they are doing.